tests/functional_tests/0013_import_test.py

```diff
@@ -61,10 +61,3 @@ def test_01_import_jail(invoke_cli, jail, skip_test, remove_file, zfs):
     )
 
     assert jail.exists is True, f'{exported_jail} jail did not import'
-
-    # Let's do a clean up on the exported files
```

Comment:
I think we should let these be in case we are running selected tests and then import would not clean up on the images dataset ? Thoughts ?

Reply:
I suppose we can, but we don't officially support running selected tests, right now it's the whole battery